lib/mbta/api/facilities.ex

```diff
@@ -0,0 +1,17 @@
+defmodule MBTA.Api.Facilities do
+  @moduledoc """
+  Responsible for fetching Stop data from the V3 API.
+  """
+
+  @mbta_api Application.compile_env!(:dotcom, :mbta_api)
```

Comment:
You'll see this pattern throughout. These modules are really just intermediaries to the `MBTA.Api` so they determine whether to use the real one or a mock. This means that we ultimately always mock `MBTA.Api.Mock.get_json` since that's all they ever use.